analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    acs_demo = pd.read_csv(os.path.join(dir,'ACS_census_tract_LA.csv' ))
</pre><p>nit: space after the commas.</p><p> </p><p>maybe we want to spell out demographics, since "demo" has a specific meaning separate from "demographics".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done. Changed the name from demo to demographics.</p>